components/integrations_marketplace/marketplace_modal.jsx

```diff
@@ -50,7 +50,8 @@ export default class MarketplaceModal extends React.Component {
         this.getMarketplacePlugins();
     }
 
-    getMarketplacePlugins = async (filter) => {
+    getMarketplacePlugins = async () => {
+        const filter = this.refs.filter ? this.refs.filter.value : '';
```

Comment:
any reason not to use `null` as opposed to `''`?

Reply:
Done